web/client/components/data/query/enhancers/geometrydetails.js

```diff
@@ -0,0 +1,24 @@
+/**
+* Copyright 2018, GeoSolutions Sas.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+const {compose, createEventHandler, defaultProps} = require('recompose');
+
+const propsStreamFactory = require('../../../misc/enhancers/propsStreamFactory');
+
+
+const dataStreamFactory = $props => {
```

Comment:
Why do we need this?

Reply:
Done to debounce the change.

Reply:
it also changes the interface to call onChangeDrawingStatus, which is not immediately clear, look at the other question a bove

Reply:
se answer above

Reply:
I mean, if you put a "transparent" debounce in the middle, it should be transparent, if I remove it the same code will work, without debouncing. We want to enhance, not to change.